android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -335,6 +335,20 @@ class BrowserToolbar @JvmOverloads constructor(
         edit.removeEditActionEnd(action)
     }
 
+    /**
+     * Hides menu button in display mode.
```

Comment:
Could you do another pass through the kDocs? It's basically just adding `the` in all of these